### ISSUE TITLE
Feature request: Control tools that are shown

### ISSUE THREAD

**Issue by oleh-andriiv** [scaleflex/filerobot-image-editor#317]
It would be really good if list of tools for display could be selected (just like with Tabs)

**Comment by AhmeeedMostafa** [comment]
Yes we will consider this in the enhancements, temporarily u could hide the tool using the css

**Comment by thijs-qv** [comment]
Is there a way to set which tool gets selected by default when clicking on a tab? The `defaultToolId` only works for the initial tab that is opened. For context: I've hidden the Text tool on the Draw tab and set the defaultToolId to Pen, which works fine when the editor loads, but when switching to another tab and then switching back to the Draw tab the Text tool gets selected even though the button is hidden.

**Comment by AhmeeedMostafa** [comment]
@thijs-qv currently not possible, maybe we could think about considering such an idea

**Comment by HimonRana** [comment]
This is a nice enhancement to have. Some tools is never used and its nice if you could hide them.
